printer/kubectl_diff.go

```diff
@@ -0,0 +1,39 @@
+package printer
+
+import (
+	"bufio"
+	"fmt"
+	"github.com/kubecolor/kubecolor/config"
+	"io"
+	"strings"
+)
+
+type DiffPrinter struct {
+	Theme *config.Theme
+}
+
+func (p *DiffPrinter) Print(r io.Reader, w io.Writer) {
+	scanner := bufio.NewScanner(r)
+	for scanner.Scan() {
+		line := scanner.Text()
+		if line == "" {
+			fmt.Fprintln(w, line)
+			continue
+		}
+		parsedLine := p.parseLine(line)
+
+		fmt.Fprintln(w, parsedLine)
+	}
+}
+
+func (p *DiffPrinter) parseLine(line string) interface{} {
+	theme := p.Theme.Diff
+	switch true {
```

Comment:
In Go you can skip the condition and it'll be equivalent to `switch true {}`

```suggestion
	switch {
```